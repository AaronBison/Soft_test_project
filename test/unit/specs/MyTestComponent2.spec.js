import MyComponent from '@/MyTestComponent2'

import { mount } from '@vue/test-utils'

test('Html_MatchesSnapshot_True', () => {  
  const wrapper = mount(MyComponent)

  expect(wrapper.html()).toMatchSnapshot()
})

test('Html_WithPassedPropToMatchesSnapshot_True', () => {  
  const wrapper = mount(MyComponent,{
    propsData: {
      value: "World!"
    }
  })

  expect(wrapper.html()).toMatchSnapshot()
})