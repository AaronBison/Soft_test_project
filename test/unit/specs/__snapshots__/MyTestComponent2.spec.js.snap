// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Html_MatchesSnapshot_True 1`] = `
<div>
    <h1>Hello Test</h1>
</div>
`;

exports[`Html_WithPassedPropToMatchesSnapshot_True 1`] = `
<div>
    <h1>Hello World!</h1>
</div>
`;
